examples/How_to_call_functions_for_knowledge_retrieval.ipynb

```diff
@@ -274,6 +186,14 @@
    "id": "dda02bdb",
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/cy/swqv6trd5_75pjyyssft8pb80000gp/T/ipykernel_54755/2080760436.py:16: DeprecationWarning: The 'Search.results' method is deprecated, use 'Client.results' instead\n",
+      "  for result in search.results():\n"
+     ]
+    },
```

Comment:
looks like search.results in deprecated. technically not part of the scope of this change, but to avoid having this error can you update to the non-deprecated method as well?